version.yml

```diff
@@ -0,0 +1 @@
+version_name: "0.1"
```

Comment:
What's this version?

Reply:
This is not defined yet properly. I will take care of this in the CD